app/frontend/src/features/auth/email/ChangeEmailPage.tsx

```diff
@@ -29,7 +30,8 @@ export default function ChangeEmailPage() {
     reset: resetForm,
   } = useForm<ChangeEmailFormData>();
   const onSubmit = handleSubmit(({ currentPassword, newEmail }) => {
```

Comment:
Would be nice to update the tests on this to check it's the sanitised input that's being called.

Reply:
I created 2 new tests for password and without password forms. If you'd rather have me edit existing tests let me know ;)

Reply:
Haha more _useful_ tests are always welcomed, but don't want to blow the scope of this too much for ones that don't have tests yet. This tests the helper as well which I much prefer than unit testing it in this case as it's only a thin wrapper around built-in functionality so we know that should work on its own already.